### ISSUE TITLE
more beautiful koma

### ISSUE THREAD

**Issue by o-jill** [o-jill/washcrus#109]
now you are using 16px x 16px png images.
but it's to ugly with high definition displays.
let's make more beautifull komas!
svg may be the best solution.

**Comment by o-jill** [comment]
this is one of the examples and it looks very easy 😄 

```svg
<?xml version="1.0"?>
<svg width="50" height="100" viewBox="0 0 50 100" version="1.1" xmlns="http://www.w3.org/2000/svg" >
 <style>
  /* <![CDATA[ */
   @import url(https://fonts.googleapis.com/earlyaccess/sawarabimincho.css);
   polygon {
    fill: #FDD667;
    stroke: black;
    stroke-width: 1px;
   }
   text {
    font-size: 40px;
    font-family: 'Sawarabi Mincho';
   }
  /* ]]> */
 </style>
 <g id="sente">
  <polygon points="25,0 45,5 50,50 0,50 5,5" />
  <text x="25" y="27.5" text-anchor="middle" alignment-baseline="central">龍</text>
 </g>
 <g id="gote" transform="translate(0,50)">
  <g transform="translate(25,25) rotate(180) translate(-25,-25)">
   <polygon points="25,0 45,5 50,50 0,50 5,5" />
   <text x="25" y="27.5" text-anchor="middle" alignment-baseline="central">龍</text>
  </g>
 </g>
</svg>
```

**Comment by o-jill** [comment]
koma_svg was made for this issue.

**Comment by o-jill** [comment]
it looks good now. dc271f8f3c

but you have to make the code easy to switch image format. not pictures them selves. only file extensions.

**Comment by o-jill** [comment]
i will support switching image format in the future.
it's another issue.

**Comment by o-jill** [comment]
merged as PR #111.
